docs/wetboek.md

```diff
@@ -604,9 +604,8 @@
 
 ### Artikel V-3 Steek- en slagwapens (27 WWM)
 
-1. Strafbaar is een persoon die een steek- of slagwapen voorhanden heeft, bij zich draagt, of opgeslagen heeft in huis of in de laadruimte van zijn of haar vervoersmiddel, waarvan redelijkerwijs kan worden aangenomen dat deze bestemd is om letsel toe te brengen.
-2. Voor steekwapens die aangemerkt kunnen worden als zijnde stiletto geldt een uitzondering; deze zijn altijd verboden.
-3. Uitgezonderd is het in het bezit zijn van een honkbalknuppel, wanneer deze niet kan worden aangemerkt als beschreven onder lid 1.
+1. Strafbaar is een persoon die een slagwapen voorhanden heeft, bij zich draagt, of opgeslagen heeft in huis of in de laadruimte van zijn of haar vervoersmiddel, waarvan redelijkerwijs kan worden aangenomen dat deze bestemd is om letsel toe te brengen.
+2. Voor alle steekwapens geldt een uitzondering; deze zijn altijd verboden.
```

Comment:
dus we gaan gewoon ff heel het beleid wat er normaal is kwa messen in  het echt niet erkennen?

Reply:
Nee, vanuit Niek & Kenneth is besloten om ALLE messen illegaal te maken.

Reply:
kan ik dit ergens terug vinden ?

Reply:
Nee, je zou het wel aan ze kunnen navragen. Het is ook al doorgekoppeld naar justitie.

Reply:
heb ff nagevraagd